docs/application/native/guides/ui/dali/touchdata.md

```diff
@@ -6,8 +6,10 @@ DALi informs you with a callback when the user touches the screen and when the t
 
 The `Dali::TouchData` class replaces the old deprecated `Dali::TouchEvent` struct. It contains functions for retrieving the information that the `Dali::TouchEvent` struct provided, but also adds the radius, touch pressure, and angle information.
 
-​        **Note**  
-        > Do not use the `Dali::TouchData` class in a container.	 As the `Dali::TouchData` class is a handle to an internal object, it must not be copied (or used in a container) as all that happens is that the handle is copied to the same object, and the internal object can change unexpectedly. If the data must be stored in the application, save only the required data (retrieved using the class functions).    
+​> **Note**  
```

Comment:
It looks like it need space. The note style is not applied.

Reply:
done